extension/lib/settings_wrapper.js

```diff
@@ -149,6 +176,18 @@ class Settings {
 		});
 	}
 
+	/*
+	Save the term document frequencies. 
+	This is a map of all words user has ever read to the number of documents they have occured in. 
+	Every time they read a new document, it adds one to each word in the dict, or adds the word to the dict
+	initialized at 1. 
+		- trackerSettings: {trackerSettingKey : termDocumentFreq}
+		Note termDocumentFreq is a dict. 
+
+	There is no "set" for this setting. This dictionary is changed using the cb function. 
```

Comment:
I'm not sure if I understand this comment.
There is setTermDocumentFreq, and it's necessary too

Reply:
Oh yes this is old, I was briefly experimenting w/ not using it, trying to update dict using a callback in the get function. Didn't work during experimentation tho; will come back to later. Nothing to understand, just mistake